src/applications/personalization/profile-2/components/connected-apps/reducers/connectedAccounts.js

```diff
@@ -0,0 +1,63 @@
+import {
```

Comment:
would be good to add tests for these at some point.

Reply:
Absolutely, I will create a ticket to add unit extensive unit tests.